catalog/app/components/Preview/loaders/useSignObjectUrls.ts

```diff
@@ -65,13 +65,15 @@ const createObjectUrlsSigner =
     return traverseUrls((idx: number): string => results[idx], jsonWithPlaceholders)
   }
 
+// Increased time to download linked data assets
+const URL_EXPIRATION = 10 * 60
```

Comment:
is this needed tho? you've already bumped the default value

Reply:
Sorry, meant to bump only this value. Forgot to remove the default or messed up when rebasing

Reply:
Yes, forgot to commit.

Though, singing only used in renderers. So, maybe, it's better to increase default value to 10 minutes, but use the old default for thumbnails and logo? What do you think?

Reply:
i think it's ok to just use the new default for everything

Reply:
Yes, it's probably the most reasonable choice. Thanks